HAM/data_scientist/assets/dataset/opener.py

```diff
@@ -0,0 +1,109 @@
+"test"
```

Comment:
this opener is the same as the data_owner one, except `shuffle=True`, is it necessary ? You could use one opener for both datasets

Reply:
The idea with this example was also to simulate having two different datasets so I used two opener even if they are similar.